clients/web/src/templates/widgets/editCollectionWidget.pug

```diff
@@ -4,10 +4,10 @@
       .modal-header
         button.close(data-dismiss="modal", aria-hidden="true", type="button") &times;
         h4.modal-title
-          if (collection)
-            | Edit collection
-          else
+          if (create)
```

Comment:
To preserve compatibility of this template, let's just change the condition from `if collection` to `if collection && collection.id`.